_includes/releases/v22.2/v22.2.6.md

```diff
@@ -0,0 +1,127 @@
+- Updated the description for [Suboptimal Insight](../v22.2/ui-insights-page.html) and added a **Learn more** link to it. [#97718][#97718]
+- Added page controls to the recent execution overview pages so that users can go to the next page when the page limit of results is reached. [#97250][#97250]
+- Fixed the pagination on the [Schema Insights view](../v22.2/ui-insights-page.html#schema-insights-view), which was incorrectly showing the number of page results. [#97641][#97641]
+
+<h3 id="v22-2-6-bug-fixes">Bug fixes</h3>
+
+- Fixed a bug that caused the server to crash if trying to restore a table from a backup generated by `BACKUP TABLE` from a schema that includes [user-defined functions](../v22.2/user-defined-functions.html), and the restore target database does not have a schema with the same name. [#96977][#96977]
+- Fixed the [`SHOW GRANTS FOR public`](../v22.2/show-grants.html) command so that it no longer returns an error that the `public` role does not exist. [#96998][#96998]
+- Fixed a bug where the [`AS OF SYSTEM TIME`](../v22.2/as-of-system-time.html) clause was handled incorrectly in an implicit transaction that had multiple statements. [#97146][#97146]
+- When upgrading from a v22.1 to v22.2.4 cluster, logging in with a new user would error with `system.privilege does not exist`. When the user privilege was checked during starting up was causing this error. Only v22.2.4 is affected. This change adds a version gate for this check. [#97183][#97183]
+- Fixed a syntax error for `SELECT ... QUERY` (without `AS`) statement. [#97156][#97156]
+- Fixed a bug where a [restore](../v22.2/restore.html) flow could hang indefinitely in the face of a context cancellation. This would manifest as a stuck restore job. [#97210][#97210]
+- Removed the following log message that was produced frequently: `lease [...] expired before being followed by lease [...]; foreground traffic may have been impacted`. [#97377][#97377]
+- Fixed a bug in the query engine that could cause incorrect results in some cases when a [zigzag join](../v22.2/joins.html) was planned. The bug could occur when the two indexes used for the zigzag join had a suffix of matching columns, but with different directions. For example, planning a zigzag join with `INDEX(a ASC, b ASC)` and `INDEX(c ASC, b DESC)` could cause incorrect results. This bug has existed since at least v19.1. It is now fixed, because the [optimizer](../v22.2/cost-based-optimizer.html) will no longer plan a zigzag join in such cases. [#97441][#97441]
+- Fixed a bug in [`ALTER TABLE ... ADD COLUMN`](../v22.2/alter-table.html#add-column) when the new column name requires quoting due to mixed case or special characters and the statement is not run in an explicit or multi-statement transaction. [#97568][#97568]
+- Fixed a bug when formatting create statements for user-defined types that require quoting, which might prevent those statements from round-tripping. [#97568][#97568]
```

Comment:
This is my favorite malapropic verbification from upstream yet: "round-tripping!" No action, from docs, needed.

Reply:
LOL